dandiapi/api/views/serializers.py

```diff
@@ -86,6 +86,63 @@ class Meta:
     # name = serializers.SlugRelatedField(read_only=True, slug_field='name')
 
 
+class DandisetVersionSerializer(serializers.ModelSerializer):
+    """The verison serializer nested within the Dandiset Serializer."""
+
+    class Meta:
+        model = Version
+        fields = [
+            'version',
+            'name',
+            'asset_count',
+            'size',
+            'status',
+            'created',
+            'modified',
```

Comment:
```suggestion
            'modified',
            'dandiset',
```

The current endpoint also returns the associated `Dandiset` nested within the `Version`.

Reply:
Including that in this endpoint is redundant, since the dandiset is at the top level anyway. I'd prefer to solve this in the CLI, since I don't see any reason to keep that in the listing endpoint.

Reply:
Oh my bad, those are e2e tests. In any case, is there a good reason to keep that redundant nesting?

Reply:
Probably not. I think the "RESTful" way to do it would be to call the `dandiset` endpoint to fetch that data instead of packing it into the version endpoint

Reply:
It's not even a restful thing, my point is that, prior to this change, this was the structure of the response:

```
"results": [
    {
        "identifier": "000001",
		...(all dandiset info),
        "draft_version": {
            "version": "draft",
            "name": "test",
            ...,
            dandiset: {
                "identifier": "000001",
                ...(all info repeated again)
            }
        }
    }
]
```

So the dandiset information is needlessly included twice. However, I totally forgot to fix the use of it in the web app, so it's a good thing your pyppeteer CI caught it. If you're okay with removing the nested `dandiset` parameter, I'll make a fix to the front-end. We should also inform the collaborators, in case the CLI makes use of it.

Reply:
Oh. Wow I didn't realize that info was literally repeated, yeah i agree that should be removed.